mozilla/fx-private-relay
# Stage fix 2022.09.07.01

This PR fixes phone fox-fooding issues caught in prod:
1. The purchase phones link needs to go directly to a plan
2. (pending) The `/phone/` route needs to be disabled if the user doesn't have the `phones` waffle flag

How to test:
1. Set your [local `phones` waffle flag](http://127.0.0.1:8000/admin/waffle/flag/) to Everyone: Unknown
2. Go to http://127.0.0.1:8000/phone/
   * [ ] You should be redirected to http://127.0.0.1:8000/premium/
3. Give your [local `phones` waffle flag](http://127.0.0.1:8000/admin/waffle/flag/) to only your local user
4. Go to http://127.0.0.1:8000/phone/
   * [ ] You should see the page to purchase phone!
5. Hover over the link to "Upgrade now"
   * [ ] It should include a new `plan=price_1Kz3rKKb9q6OnNsLkiBHTGXE` url param
